python strongnumber.py

```diff
@@ -0,0 +1,21 @@
+def strongnumber(n):
+    sum=0
```

Comment:
Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279935)